modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="update-mirror-repository-oc-mirror_{context}"]
+= Mirroring resources using the oc-mirror plug-in
+
+Using the oc-mirror OpenShift CLI (`oc`) plug-in is the recommended method for mirroring images onto a mirror registry. This can be useful in several situations, including fully or partially disconnected environments.
+
+.Procedure
+
+. Navigate to _Mirroring images for a disconnected installation using the oc-mirror plug-in_.
```

Comment:
I'd recommend adding the "the <x> page" or "the <x> section" so that people know what you're talking about.

```suggestion
. Navigate to the _Mirroring images for a disconnected installation using the oc-mirror plug-in_ section of the documentation.
```